src/components/bb/QuestionForm.vue

```diff
@@ -29,13 +29,13 @@ export default {
       this.questions = questions;
     },
     addQuestion() {
-      const header = getHeader();
+      const headers = getHeader();
       const url = 'csc/questions/create';
-      axios.post(url, {}, header)
```

Comment:
Oh hmm why is this unnested headers

Reply:
I only updated `header` to `headers`. I left everything else the same. You guys need to check if axios expects nested or unnested. I have been testing only on the csc/questions/generate endpoint and related frontend functions.

Reply:
Yes i asking myself